### ISSUE TITLE
aria-actions attribute

### ISSUE THREAD

**Issue by janewman** [mozilla/standards-positions#1422]
### Specification title

aria-actions addition to the ARIA spec

### Specification or proposal URL (if available)

https://github.com/w3c/aria/pull/1805

### Proposal author(s)

@smhigley

### web-feature id

None yet, filed: https://github.com/web-platform-dx/web-features/issues/4108

### Bugzilla URL

https://bugzilla.mozilla.org/show_bug.cgi?id=1992029

### Mozillians who can provide input

@jcsteh

### WebKit standards-position

https://github.com/WebKit/standards-positions/issues/686

### Other information

A new aria attribute that references a list of elements that should be exposed as secondary actions.

**Comment by jcsteh** [comment]
This was mostly shipped in Firefox 146, with the UIA mappings shipped in 148.

**Comment by jcsteh** [comment]
Retrospectively: I'm proposing a position of positive. We believe this is important in order for assistive technology users to be able to efficiently and consistently discover and activate secondary actions, especially in complex web apps. This functionality has already been available to native mobile apps for quite some time and it is very popular among assistive technology users.
